src/ui/tabview/content/SettingsContent.qml

```diff
@@ -0,0 +1,213 @@
+/*
+ * This file is part of Liri Browser
+ *
+ * Copyright (C) 2016 Tim Süberkrüb <tim.sueberkrueb@web.de>
+ *
+ * $BEGIN_LICENSE:GPL3+$
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * $END_LICENSE$
+*/
+
+import QtQuick 2.7
+import QtQuick.Layouts 1.1
+import QtQuick.Controls 2.0
+import Fluid.Core 1.0
+import Fluid.Controls 1.0
+import core 1.0
+import "../.."
+
+TabContent {
+    id: content
+
+    Flickable {
+        anchors {
+            fill: parent
+            margins: 16
```

Comment:
Done in https://github.com/lirios/browser/pull/17/commits/95513c7897debff95de4199985f2ff8166e8812a